charts/hivemq-platform-operator/templates/deployment.yml

```diff
@@ -96,15 +98,15 @@ spec:
               value: "{{ .Values.http.port }}"
             - name: quarkus.http.ssl-port
               value: "{{ .Values.https.port }}"
-            {{- if and (ne .Values.tls.secretName "nil") (.Values.tls.secretName) }}
+            {{- if .Values.tls.secretName }}
```

Comment:
Does `if` ignore `nil` or do we need `with` here? Same with `keystorePassword`, `truststorePassword` and `secretName`?

Reply:
Same comment as per https://github.com/hivemq/helm-charts/pull/318/files#r1742230911

Reply:
Will put them back. Good catch!